.github/pull_request_template.md

```diff
@@ -1,26 +1,66 @@
 # Pull Request
 
 ## Description
-<!-- A brief description of what this PR does -->
+
+<!-- Provide a high-level, general description of the PR that can be used in release notes if needed. -->
+
+<!-- If applicable, provide screenshots or GIFs of the changes. -->
 
 ## Related Issues
-<!-- List any related issues or tickets -->
+
+<!-- List the user story, task, or bug in ADO that this PR addresses -->
 
 ## Changes
-<!-- List the key changes in this PR -->
+
+<!-- Describe in detail all changes included in this PR. -->
 
 - Change 1
 - Change 2
-- Change 3
 
 ## Testing
-<!-- Describe the tests that were run or any QA steps that were taken -->
 
-- Test 1
-- Test 2
+### Unit Tests
+
+<!-- Describe the unit tests changed or added with this PR. -->
+
+- Unit Test 1
+- Unit Test 2
+
+### Manual Testing
+
+<!-- Describe precisely what manual testing you have conducted on this PR. -->
+
+- Manual Test 1
+- Manual Test 2
+
+## Checklist 
+
+<!-- If you are not sure about any of the following items, mention it to your reviewer. -->
+
+### General
+
+- [ ] The PR provides a clear high-level description for a general audience, including photos or GIFs if applicable
+- [ ] The PR includes a detailed change list
+- [ ] The PR is associated with a bug, user story, or other work item on Azure DevOps
+- [ ] Changes to existing functionality have been communicated to affected users (including other DataHub team members), or there is a plan to communicate to affected users, or not applicable
+
+### Code Quality
+
+- [ ] The code is easily understandable and maintainable
+- [ ] The code is consistent with .NET standards and the project architecture
+- [ ] The code is adequately documented (such as in-code comments or dev documentation)
+- [ ] The code does not contain duplicate code that could be abstracted
+
+### Tests
+
+- [ ] New code behaves as expected and achieves its intended purpose
+- [ ] Existing functionality behaves as expected and has been tested
+- [ ] The code has been tested in both English and French, if applicable
+- [ ] The PR includes adequate unit tests to cover code changes
+- [ ] The PR does not break existing tests
 
-## Checklist
-<!-- Ensure the following tasks are complete -->
+### Additional Requirements
 
-- [ ] Code follows dotnet coding standards
-- [ ] Tests added/updated to cover changes
+- [ ] The code does not introduce any security vulnerabilities
+- [ ] The PR includes updated localization files or makes no changes that impact localization
+- [ ] The PR follows accessibility requirements or makes no changes that impact accessibility
```

Comment:
See my comments in the review template - but I think this should be more of a checklist of things like

- I made unit tests that cover the changes
- I added localization
- I wrote up the details

or along those lines - with the goal of giving some context for reviewers before they start. Maybe a checkbox or short list for each of the subreview sections?